test/Microsoft.Extensions.Logging.AzureAppServices.Test/AzureDiagnosticsConfigurationProviderTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Extensions.Logging.AzureAppServices.Internal;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Extensions.Logging.AzureAppServices.Test
+{
+    public class AzureDiagnosticsConfigurationProviderTests
+    {
+        [Fact]
+        public void NoConfigFile()
+        {
+            var tempFolder = Path.Combine(Path.GetTempPath(), "AzureWebAppLoggerThisFolderShouldNotExist");
+
+            var contextMock = new Mock<IWebAppContext>();
+            contextMock.SetupGet(c => c.HomeFolder)
+                .Returns(tempFolder);
+
+            var config = SiteConfigurationProvider.GetAzureLoggingConfiguration(contextMock.Object);
+
+            Assert.NotNull(config);
+        }
+
+        [Fact]
+        public void ReadsSettingsFileAndEnvironment()
```

Comment:
Would be good to add a test the ensure the ordering of env vs file is correct when there are matching config keys in both. settings.json takes precedence over environment, right?